network.go

```diff
@@ -74,6 +74,7 @@ type svcInfo struct {
 	svcMap     map[string][]net.IP
 	svcIPv6Map map[string][]net.IP
 	ipMap      map[string]string
+	ports      map[string][]servicePorts
```

Comment:
Why do we need a slice of  `servicePorts` as value to the map? It looks `servicePorts` itself has a list of containers/ports